src/globals.js

```diff
@@ -0,0 +1,29 @@
+// The two following arrays are the same size, and any particular
+// word has the same index in both
+//
+// Array of displayed word spans
+const wordSpanArray = []
+// Array of sanitised strings to compare user input to
+const comparisonWordsArray = []
+
+// Span of the current word
+let currentWordSpan = null
+// User input comparison string for the current word
+let comparisonWord = null
+
+// to keep track of score
+let totalCorrectWords = 0
+let totalIncorrectWords = 0
+
+// for timer
+let countdown = 59
+const timer = document.querySelector(".timer-text")
+let countdownStarted = false
+
+// results
+let gameOver = false
+const resultsContainer = document.querySelector("#results-popup")
```

Comment:
These are fine because they're in the HTML I guess. Just a note that querySelecting should generally be done at the moment you need the thing because in a dynamic document the DOM object might go out of date (e.g. if you querySelected currentWordSpan at the start)